aten/src/ATen/native/mps/operations/Copy.mm

```diff
@@ -267,7 +252,7 @@ void copy_cast_mps(at::Tensor& dst, const at::Tensor& src,
 
   auto storage_byte_offset = src_.storage_offset() * src_.itemsize();
   id<MTLBuffer> sourceBuffer = __builtin_bit_cast(id<MTLBuffer>, src_.storage().data());
-  if (!src_.is_contiguous()) {
+  if (src_.is_view()) {
```

Comment:
I think we don't want to enable this is_view() check yet - a gather might be slower than a direct blit here

Reply:
Yes currently I don't see any test-case or a situation where we need to call gather , than a simple blit.

Reply:
Denis recommended this in Slack. I can remove it.